ddtrace/internal/remoteconfig/client.py

```diff
@@ -192,7 +192,7 @@ def __init__(self):
         self._client_tracer = dict(
             runtime_id=runtime.get_runtime_id(),
             language="python",
-            tracer_version=ddtrace.__version__,
+            tracer_version=ddtrace.__version__.replace("rc", "-rc", 1),
```

Comment:
probably worth having a comment about RCM requiring semver compliant versions and how our version isn't necessarily semver